gateway-core/components/org.wso2.carbon.gateway.core/src/main/java/org/wso2/carbon/gateway/core/flow/mediators/builtin/invokers/CallMediator.java

```diff
@@ -39,8 +43,8 @@
 
     private static final Logger log = LoggerFactory.getLogger(CallMediator.class);
     private String outboundEPKey;
-    private String integration;
-    private String inputMessageParameter;
+    private String integrationKey;
```

Comment:
can we explain what are these variables ( usage, why it exists etc etc) in a java - doc.